competitions/Climate2030/etl/deploy

```diff
@@ -6,21 +6,22 @@
 # it under the terms of the GNU Affero General Public License as published by
 # the Free Software Foundation, either version 3 of the License, or
 # (at your option) any later version.
-# 
```

Comment:
I'm not even sure how these came about (my editorconfig removes trailing whitespace so I suspect that's the issue -- since that block is copy pasted I bet there is a trailing whitespace so every time I save in a new `deploy` it is going to remove that!)

Reply:
These are in their own commit now (thank you https://gist.github.com/slifty/68bd451896f6494b870280c00dc4fe9c !)